build.gradle.kts

```diff
@@ -30,33 +30,53 @@ repositories {
         }
     }
     mavenCentral()
+    maven("file://${layout.projectDirectory.dir("components/develocity").asFile.path}")
 }
 
 val isDevelopmentRelease = !hasProperty("finalRelease")
 val releaseVersion = releaseVersion()
 val releaseNotes = releaseNotes()
 val distributionVersion = distributionVersion()
-val buildScanSummaryVersion = "0.9-2024.1.2-20240515173555"
+val buildScanSummaryVersion = "0.9-2024.1.2-prerelease"
 
 allprojects {
     version = releaseVersion.get()
 }
 
 val argbash by configurations.creating
 val mavenComponents by configurations.creating
+val buildScanSummaryComponent by configurations.creating {
+    attributes.attribute(
+        TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE,
+        project.objects.named(TargetJvmEnvironment.STANDARD_JVM)
+    )
+}
 
 dependencies {
     argbash("argbash:argbash:2.10.0@zip")
     mavenComponents(project(":configure-gradle-enterprise-maven-extension"))
     mavenComponents("com.gradle:gradle-enterprise-maven-extension:1.18.4")
     mavenComponents("com.gradle:common-custom-user-data-maven-extension:1.13")
+    buildScanSummaryComponent("com.gradle.develocity:build-scan-summary:$buildScanSummaryVersion")
```

Comment:
Should the group be `com.gradle` like the other things we publish? I see the `build-scan-summary` is published under the `com.gradle.develocity` group too.

Reply:
Done